elftools/elf/elffile.py

```diff
@@ -12,6 +12,8 @@
 
 try:
     import resource
+    if "getpagesize" not in dir(resource):
```

Comment:
I agree it would be simpler to catch AttributeError.

BTW hasattr(resource, "getpagesize") would have been a better than using dir().